docs/griptape-cloud/structures/create-structure.md

```diff
@@ -0,0 +1,16 @@
+# Structures
+
+Structures are the primary building block in the Griptape.
+
+## Create a Structure
+
+1. [Connect Your GitHub Account](https://cloud.griptape.ai/account) in your Griptape Cloud account
+1. Install the [Griptape Cloud GitHub app](https://github.com/apps/griptape-cloud/installations/new/) to any account or organization you'd like to pull code in from
+    - Be sure to allow the app access to `All Repositories` or select the specific repositories you need
+1. Create a [Structure Config YAML](structure-config.md) in your repository
+
+You can now [create a Structure](https://cloud.griptape.ai/structures/create) by providing your GitHub repository information.
```

Comment:
this section is a good example of where it can be confusing as to where our links go to. We're taking them into the Cloud account, into GitHub, and into the documentation all in one series of instructions.

Reply:
This section I would say is the most obvious based on context. Do you have suggestions on how to make it more clear? I can add more words to each hyperlink, but I don't want to remove the links.

Reply:
per discussion, I think we can be a little pedantic for where it links to with some wording (e.g., "You can now [create a Structure in Griptape Cloud console]" or "Follow the [Documentation on Structure Config YAMLs] to create one...").